src/sentry/static/sentry/app/bootstrap.jsx

```diff
@@ -0,0 +1,151 @@
+import 'bootstrap/js/alert';
```

Comment:
This is pretty much 100% just a rename from app/index.jsx, it just needed to be a diff module